fusion/lidar3_single[] lidar_points
float64 stamp
